Framework/Algorithms/src/GetDetectorOffsets.cpp

```diff
@@ -208,7 +246,7 @@ double GetDetectorOffsets::fitSpectra(const int64_t s) {
   // Return if peak of Cross Correlation is nan (Happens when spectra is zero)
   // Pixel with large offset will be masked
   if (std::isnan(peakHeight))
-    return (1000.);
+    return (DBL_MAX);
```

Comment:
It may be better to convert this to a (file?) constant
```C++
constexpr double FIT_FAILURE{DBL_MAX};
```
to make it easier for future readers to see